mmdet/optimizer/mmsgd.py

```diff
@@ -0,0 +1,84 @@
+import torch
+from torch.optim.optimizer import Optimizer, required
+
+from .registry import OPTIMIZERS
+
+
+@OPTIMIZERS.register_module
+class MMSGD(Optimizer):
```

Comment:
It can be just named `CopyOfSGD` and derived from `torch.optim.SGD`. We may also document how to use custom optimizers.